resources/controlplane-deployment.toml

```diff
@@ -1,273 +1,282 @@
-
-  #[apim.workflow]
-  #enable = false
-  #service_url = "https://localhost:9445/bpmn"
-  #username = "$ref{super_admin.username}"
-  #password = "$ref{super_admin.password}"
-  #callback_endpoint = "https://localhost:${mgt.transport.https.port}/api/am/admin/v0.17/workflows/update-workflow-status"
-  #token_endpoint = "https://localhost:${https.nio.port}/token"
-  #client_registration_endpoint = "https://localhost:${mgt.transport.https.port}/client-registration/v0.17/register"
-  #client_registration_username = "$ref{super_admin.username}"
-  #client_registration_password = "$ref{super_admin.password}"
-
-  #data bridge config
-  #[transport.receiver]
-  #type = "binary"
-  #worker_threads = 10
-  #session_timeout = "30m"
-  #keystore.file_name = "$ref{keystore.tls.file_name}"
-  #keystore.password = "$ref{keystore.tls.password}"
-  #tcp_port = 9611
-  #ssl_port = 9711
-  #ssl_receiver_thread_pool_size = 100
-  #tcp_receiver_thread_pool_size = 100
-  #ssl_enabled_protocols = ["TLSv1","TLSv1.1","TLSv1.2"]
-  #ciphers = ["SSL_RSA_WITH_RC4_128_MD5","SSL_RSA_WITH_RC4_128_SHA"]
-
-  #[apim.notification]
-  #from_address = "APIM.com"
-  #username = "APIM"
-  #password = "APIM+123"
-  #hostname = "localhost"
-  #port = 3025
-  #enable_start_tls = false
-  #enable_authentication = true
-
-  #[apim.token.revocation]
-  #notifier_impl = "org.wso2.carbon.apimgt.keymgt.events.TokenRevocationNotifierImpl"
-  #enable_realtime_notifier = true
-  #realtime_notifier.ttl = 5000
-  #enable_persistent_notifier = true
-  #persistent_notifier.hostname = "https://localhost:2379/v2/keys/jti/"
-  #persistent_notifier.ttl = 5000
-  #persistent_notifier.username = "root"
-  #persistent_notifier.password = "root"
-
-  [[event_handler]]
-  name="userPostSelfRegistration"
-  subscriptions=["POST_ADD_USER"]
-
-  [service_provider]
-  sp_name_regex = "^[\\sa-zA-Z0-9._-]*$"
-
-  [[event_listener]]
-  id = "token_revocation"
-  type = "org.wso2.carbon.identity.core.handler.AbstractIdentityHandler"
-  name = "org.wso2.is.notification.ApimOauthEventInterceptor"
-  order = 1
-  [event_listener.properties]
-  notification_endpoint = "https://localhost:${mgt.transport.https.port}/internal/data/v1/notify"
-  username = "${admin.username}"
-  password = "${admin.password}"
-  'header.X-WSO2-KEY-MANAGER' = "default"
-
-  [transport.https.properties]
-  proxyPort = 443
-
-  {{ if .Values.wso2.deployment.persistentRuntimeArtifacts.apacheSolrIndexing.enabled }}
-  [database.local]
-  url = "jdbc:h2:/home/wso2carbon/solr/database/WSO2CARBON_DB;DB_CLOSE_ON_EXIT=FALSE"
-
-  [indexing]
-  location = "/home/wso2carbon/solr/indexed-data"
-  {{ else }}
-  [database.local]
-  url = "jdbc:h2:./repository/database/WSO2CARBON_DB;DB_CLOSE_ON_EXIT=FALSE"
-  {{ end }}
+hostname = "{{ .Values.wso2.deployment.am.ingress.management.hostname }}"
+#offset=0
+base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+#discard_empty_caches = false
+server_role = "default"
```

Comment:
This should be control plane.

Reply:
Since this is used for tryout, we agreed to go with "default" server role